receiver/otelarrowreceiver/config.go

```diff
@@ -14,17 +14,12 @@ import (
 
 // Protocols is the configuration for the supported protocols.
 type Protocols struct {
-	GRPC  configgrpc.ServerConfig `mapstructure:"grpc"`
-	Arrow ArrowConfig             `mapstructure:"arrow"`
+	GRPC      configgrpc.ServerConfig `mapstructure:"grpc"`
+	Arrow     ArrowConfig             `mapstructure:"arrow"`
+	Admission AdmissionConfig         `mapstructure:"admission"`
```

Comment:
On second thought, maybe "admission" belongs one level up, as it's not a protocol. It would go in the `Config` struct.